app/controllers/plans_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class PlansController < ApplicationController
+  def index
+    @plans = Plan.all
+  end
+
+  def create
+    @plan = Plan.new(plan_params)
+    Rails.logger.debug @plans
+    if @plan.save
+      flash[:notice] = 'Plan Successfully Created.'
+      redirect_to @plan
+    else
+      flash[:notice] = 'Failed to create Plan.'
```

Comment:
this should be flash[:error] and flash[:success] instead of notice and style them accordingly. Explore this more